src/app/page.js

```diff
@@ -1,40 +1,119 @@
+            justifyContent: 'end'
+          }}
+        >
+          <Pie
+            data={[
+              {id: 'surface', value: stats.activPointsSurfaceCount, label: 'Surface'},
+              {id: 'souterrain', value: stats.activPointsSouterrainCount, label: 'Souterrain'}
+            ]}
+          />
+        </Box>
+      </Box>
+      <Box className='m-auto py-8' sx={{maxWidth: '1200px'}}>
+        <Typography variant='body1' className='p-5'>
+          Il s’agit d’un outil d’aide à la décision, <b>destiné à faciliter l’évaluation et le suivi dans le temps des impacts des prélèvements sur les milieux.</b> Il s’adresse aussi bien aux services en charge de l’instruction et du contrôle des autorisations administratives qu’aux préleveurs bénéficiaires de ces autorisations. Les établissements publics, collectivités ou encore le grand public y trouveront également des informations utiles à la compréhension et la mise en œuvre d’une gestion globale et concertée de la ressource en eau, telle que prévue par le Schéma directeur d’aménagement et de gestion des eaux (SDAGE) de La Réunion.
+        </Typography>
+        <Typography variant='body1' className='p-5'>
+          Basé sur une cartographie qui se veut exhaustive, l’outil décrit les <b>modalités d’exploitations des points de prélèvement</b> en activité ou passés (bénéficiaires, autorisations délivrées, volumes autorisés et valeurs seuils à respecter…) et valorise les <b>données de suivi collectées en continu</b>.
+        </Typography>
+        <Typography variant='body1' className='p-5'>
+          Depuis 2024, ces dernières sont transmises mensuellement par les préleveurs via un formulaire en ligne (<a href='https://www.demarches-simplifiees.fr/commencer/suivi-prelevements-eau-974'>https://www.demarches-simplifiees.fr/commencer/suivi-prelevements-eau-974)</a>. A ce jour, tous les prélèvements sont concernés sauf ceux soumis au régime des installations classées pour la protection de l’environnement (ICPE) ou des concessions hydroélectriques.
+        </Typography>
+        <Typography variant='body1' className='p-5'>
+          Identifié par le SDAGE 2022-2027, le projet « suivi des prélèvements d’eau » est porté par la DEAL de La Réunion depuis 2022, en lien étroit avec ses partenaires locaux (ARS, Office de l’eau, BRGM). Il s’appuie sur un travail de longue haleine de structuration des données. Depuis 2023, il bénéficie de l’appui financier de la <a href='https://beta.gouv.fr/incubateurs/mtes.html'>Fabrique numérique</a>, l’incubateur de « startups d’Etat » du ministère en charge de l’écologie.
+        </Typography>
+      </Box>
+      <Box
+        className='w-full'
+        sx={{
+          backgroundColor: fr.colors.decisions.background.alt.blueFrance.default,
+          p: '3em'
+        }}
+      >
+        <Box
+          className='flex flex-wrap justify-between items-center'
+          sx={{
+            maxWidth: '1000px',
+            m: 'auto'
+          }}
+        >
```

Comment:
On a une marge qui est différente des autres sections ici.

![Capture d’écran 2025-03-11 à 10 05 18](https://github.com/user-attachments/assets/ea5a3447-71a9-4b69-a56c-57683a1096cb)


```suggestion
      <Box
        sx={{
          backgroundColor: fr.colors.decisions.background.alt.blueFrance.default,
          p: '3em'
        }}
      >
        <Box
          className='fr-container flex flex-wrap justify-between items-center'
        >
 ```

Reply:
Oui, c'était voulu, mais je peux ajouter du padding pour replacer comme avant.
Je n'aime pas trop quand tout est aligné "bêtement".